libs/blocks/merch-card/merch-card.js

```diff
@@ -117,27 +119,63 @@ const getPodType = (styles) => styles?.find((style) => CARD_TYPES.includes(style
 const isHeadingTag = (tagName) => /^H[2-5]$/.test(tagName);
 const isParagraphTag = (tagName) => tagName === 'P';
 
+function transformElement(element) {
```

Comment:
it looks like this is used only once and its purpose is not clear, I suggest to keep the logic inline where it is called.

Reply:
Removed